subprojects/dependency-management/src/main/java/org/gradle/internal/resource/transfer/ResourceOperation.java

```diff
@@ -59,7 +61,7 @@ public void logProcessedBytes(long processedBytes) {
         long processedKB = totalProcessedBytes / 1024;
         if (processedKB > loggedKBytes) {
             loggedKBytes = processedKB;
-            final String progressMessage = String.format("%s/%s %sed", getLengthText(totalProcessedBytes), contentLengthString, operationType);
+            final String progressMessage = String.format("%s/%s %sed: %s", getLengthText(totalProcessedBytes), contentLengthString, operationType, resourceName);
```

Comment:
A simple concatenation would probably be more efficient if this is called a lot.